src/Components/WebView/Platforms/WebView2/src/PublicAPI.Shipped.txt

```diff
@@ -0,0 +1 @@
+﻿
```

Comment:
Copy from eng/PublicAPI.empty.txt instead of letting the analyzer create these files. We want the `#nullable enable` bit from the get-go.